### PR TITLE
Fix requirements for DLC setting tensorflow to be a subpackage

### PR DESCRIPTION
**Pull request by sidhulyalkar**
- Fixed typo in docstring

### CODE REVIEW

**Review by kabilar — CHANGES_REQUESTED** [review]
Thanks @sidhulyalkar!

Please:
- [ ] Update version.py
- [ ] Update changelog.md
- [ ] Make a new package release once this pull request is merged.

**Comment by CBroz1** [comment]
@sidhulyalkar Please also confirm that this installation works in a clean conda environment. It occurs to me that, by specifying the extras here, pip may resolve to only the latest version, which may or dependency conflicts with other items in the worker's environment

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,9 @@
 Observes [Semantic Versioning](https://semver.org/spec/v2.0.0.html) standard and 
 [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) convention.
 
+## [0.2.2] - 2022-12-16
++ Update - Requirements to accomodate DLC package requirements changing
++ Fix - Typos in docstrings
```
Comment (CBroz1) [inline CHANGELOG.md:8]: Please follow the style in the rest of the doc. The [markdown lint](https://github.com/DavidAnson/markdownlint) extension can automate suggestions. 

```suggestion

## [0.2.2] - 2022-12-16

+ Update - Requirements to accomodate DLC package requirements changing
+ Fix - Typos in docstrings

```

**Comment by sidhulyalkar** [comment]
> @sidhulyalkar Please also confirm that this installation works in a clean conda environment. It occurs to me that, by specifying the extras here, pip may resolve to only the latest version, which may or dependency conflicts with other items in the worker's environment

Yes, confirmed that the installation works, with tensorflow being installed properly in a new conda environment along with rest of the dependencies.

File: CHANGELOG.md (hunk 1 above)
Comment (sidhulyalkar) [reply CHANGELOG.md]: > Please follow the style in the rest of the doc. The [markdown lint](https://github.com/DavidAnson/markdownlint) extension can automate suggestions.

Updated it after installing the markdown lint extension, it's quite useful thanks!

File: element_deeplabcut/version.py [hunk 2]
```diff
@@ -1,4 +1,4 @@
 """
 Package metadata
 """
-__version__ = "0.2.1"
+__version__ = "0.2.2"
```
Comment (kabilar) [inline element_deeplabcut/version.py:4]:
```suggestion
__version__ = "0.2.1"
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,11 @@
 Observes [Semantic Versioning](https://semver.org/spec/v2.0.0.html) standard and 
 [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) convention.
 
+## [0.2.2] - 2022-12-16
+
++ Update - Requirements to accomodate DLC package requirements changing
++ Fix - Typos in docstrings
+
 ## [0.2.1] - Unreleased
```
Comment (kabilar) [inline CHANGELOG.md:12]:
```suggestion
## [0.2.1] - 2022-12-16

+ Update - Requirements to accomodate DLC package requirements changing
+ Fix - Typos in docstrings
```

**Review by CBroz1 — APPROVED** [review]

**Review by kabilar — APPROVED** [review]

**Review by kabilar — APPROVED** [review]

**Pull request merged**